### DISCUSSION TITLE
Multiple Ventoy

### DISCUSSION THREAD

**Discussion by Kebab-cmd** [ventoy/Ventoy#927] [Q&A]
I´ve tried a few things but i´m stuck. I´m trying to get a 500GB Ext SSD to work with Legacy, due to the 137GB Limitation i would have round about 350GB Free Space Left, am i able to reuse this for another bootable version of Ventoy for Uefi use? My Goal is both Uefi and Legacy Support, so when im booting Legacy it access the first 150GB and when im booting uefi the remaining 350GB

**Comment by steve6375** [comment]
Even if you manage to make a Legacy+UEFI USB drive, you will find that some (buggy) BIOSes will only UEFI-boot from the USB drive as they do not provide the option to Legacy boot as long as there are UEFI boot files present on the USB drive. This means that on those particular systems, you cannot boot to Legacy-only ISOs from your dual-boot USB drive.
The simplest solution is to use two Ventoy USB drives. One <137GB for Legacy (with the \EFI folder deleted on the 2nd partition) and another for UEFI (full UEFI+MBR ventoy - legacy partitions or GPT partitions).
Of course, you can use one drive (just format the first partition and copy on up to 137GB of Legacy payload files first - then add the UEFI payloads) but you won't be able to add another legacy ISO without reformatting!
The other alternative is to use a different Legacy+UEFI multiboot solution...
